examples/wallet-dapi-fromHDPubKey.js

```diff
@@ -13,8 +17,8 @@ const account = wallet.getAccount();
 
 
 const start = async () => {
-  console.log('Balance Conf', await account.getBalance(false, false));
-  console.log('Balance Unconf', await account.getBalance(true, false));
+  console.log('Balance Conf', await account.getConfirmedBalance(false, false));
+  console.log('Balance Unconf', await account.getUnconfirmedBalance(true, false));
```

Comment:
These two should have only one parameter, `displayDuffs`.

Reply:
Thanks ! Fixed.